shared/javascripts/local_storage.js

```diff
@@ -87,14 +87,22 @@ var ls = {
   }
 };
 
-// Determine whether localstorage can be used directly
```

Comment:
The try block should have a comment indicating that it is being used to test which platform it is running on and selecting the appropriate storage driver.